lib/generators/module/templates/README.md

```diff
@@ -0,0 +1,28 @@
+# Vsp
+Short description and motivation.
+
+## Usage
+How to use my plugin.
+
+## Installation
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'vsp'
```

Comment:
Can you rename this file `README.md.erb` and insert the actual name of the gem here with the path? 
e.g. `gem 'appeals_api', path: 'modules/appeals_api'`